pkg/proto/core/v1/util.go

```diff
@@ -0,0 +1,201 @@
+package corev1
+
+import (
+	"github.com/rs/zerolog/log"
+	"google.golang.org/protobuf/proto"
+
+	v0 "github.com/authzed/authzed-go/proto/authzed/api/v0"
+)
+
+//revive:disable
+
+// Mnually created to work around the following issue:
+// https://github.com/envoyproxy/protoc-gen-validate/issues/481
+var _Metadata_MetadataMessage_InLookup = map[string]struct{}{
```

Comment:
Copied from authzed-go client to preserve the gen-validate workaround

Reply:
Should probably be in a `core.pb.validate.manual.go` or somesuch